clio.tests/ApplicationInstallerTests.cs

```diff
@@ -44,6 +44,45 @@ public void RestartApplicationAfterInstallPackageInNet6() {
 	}
 
 
+	[Test]
+	public void InstallWithOptionsCallsInstallWithOptionsWhenFailOnWarningIsTrue() {
```

Comment:
When creating tests, make sure to add Description attrubute explainign the test. For example
`[Test, Description("This test checks that the InstallWithOptions method can be called with FailOnWarning set to true.")]`


I want to see tests broken down into AAA style with clear separation between
//Arrange

//Act

//Assert